src/components/app/list/AppListView.tsx

```diff
@@ -108,43 +115,67 @@ export class AppListView extends Component<AppListViewProps>{
         }
     }
 
+    renderGuidedCards() {
+        return (
+            <div className="devtron-app-guided-cards-container">
+                <h2 className="fs-24 fw-6 lh-32 m-0 pt-40 dc__align-center">Create your first application</h2>
+                <div className="devtron-app-guided-cards-wrapper">
+                    <ContentCard
+                        redirectTo={DEVTRON_NODE_DEPLOY_VIDEO}
+                        isExternalRedirect={true}
+                        imgSrc={NodeAppThumbnail}
+                        title={HELM_GUIDED_CONTENT_CARDS_TEXTS.WatchVideo.title}
+                        linkText={HELM_GUIDED_CONTENT_CARDS_TEXTS.WatchVideo.linkText}
+                        LinkIcon={PlayMedia}
+                        linkIconClass="scb-5 mr-8"
+                        linkIconPlacement={CardLinkIconPlacement.BeforeLink}
+                    />
+                    <ContentCard
+                        redirectTo={`${URLS.APP}/${URLS.APP_LIST}/${AppListConstants.AppType.DEVTRON_APPS}/${AppListConstants.CREATE_DEVTRON_APP_URL}`}
+                        imgSrc={DeployCICD}
+                        title={HELM_GUIDED_CONTENT_CARDS_TEXTS.StackManager.title}
+                        linkText={HELM_GUIDED_CONTENT_CARDS_TEXTS.StackManager.createLintText}
+                        LinkIcon={ArrowRight}
+                        linkIconClass="scb-5"
+                        linkIconPlacement={CardLinkIconPlacement.AfterLinkApart}
+                    />
+                </div>
+            </div>
+        )
+    }
 
     render() {
         if (this.props.view === AppListViewType.LOADING) {
-            return <React.Fragment>
+            return (
                 <div className="dc__loading-wrapper">
                     <Progressing pageLoader />
                 </div>
-            </React.Fragment>
-        }
-
-        else if (this.props.view === AppListViewType.EMPTY) {
-            return <React.Fragment>
-                <DevtronAppGuidePage openDevtronAppCreateModel={this.props.openDevtronAppCreateModel} />
-            </React.Fragment>
-        }
-
-        else if (this.props.view === AppListViewType.NO_RESULT) {
-            return <React.Fragment>
-                <Empty view={this.props.view}
-                    title={"No apps found"}
+            )
+        } else if (this.props.view === AppListViewType.EMPTY) {
+            return this.renderGuidedCards()
+        } else if (this.props.view === AppListViewType.NO_RESULT) {
+            return (
+                <Empty
+                    view={this.props.view}
+                    title={'No apps found'}
                     message={"We couldn't find any matching applications."}
```

Comment:
`message={"We couldn't find any matching applications."}` Can Extract this messaging in constant as well

Reply:
Done

Reply:
Also, moved some other consts & made some related changes.